### DISCUSSION TITLE
Leave items in the `Resolve merge conflicts` window after a merge to be able to continue merge or edit them

### DISCUSSION THREAD

**Discussion by andry81** [gitextensions/gitextensions#10985] [Q&A] [answered]
### Feature description

The `Resolve merge conflicts` window forgets each merge conflict after merge. But sometimes there is a need to continue merge the last or an arbitrary conflict.

Could you do not delete merged items from the list and just checks them instead as merged to be able to reopen them for continue a merge?

### Environment

- Git Extensions 4.1.0.16698
- Build 1fe52a137acef1fbd2b653dea263b44ad5844177
- Git 2.40.1.windows.1
- Microsoft Windows NT 6.1.7601 Service Pack 1
- .NET 6.0.16
- DPI 96dpi (no scaling)
- Portable: False
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 6.0.10 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.16 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

**Comment by RussKie** [comment]
Please use the "Commit" button if you wish to modify the files:
![image](https://github.com/gitextensions/gitextensions/assets/4403806/2d4efef1-2b67-49f1-80e1-5e48ba76f528)

**Comment by andry81** [comment]
This is window for an interactive rebase. I was talking about the merge window. There is no such a button.

**Comment by mstv** [comment, marked as answer]
You can close the "Resolve conflicts" dialog, use GE as usual, and re-open the dialog using the banner in the revision grid.

![image](https://github.com/gitextensions/gitextensions/assets/36601201/f235ff72-aa29-4cd4-9a6f-1b8d8b5f4538)

![image](https://github.com/gitextensions/gitextensions/assets/36601201/2e60ee7c-56e7-4962-a48c-025d2faeaa84)

---

> Could you do not delete merged items from the list and just checks them instead as merged to be able to reopen them for continue a merge?

Since the GE windows are filled with the output from Git commands, GE does not delete them, but they rather disappear from the Git output. I.e. it could become error-prone to keep a list in GE in addition and to open the diff or merge tool without having the three conflict files anymore. This might be a useful feature - for me, too. Though I guess it would be too much work and too much code.
